test/configs/zombieDancelightSingleContainer.json

```diff
@@ -0,0 +1,108 @@
+                                "scheduling_lookahead": 2,
+                                "num_cores": 4
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "nodes": [
+            {
+                "name": "alice",
+                "ws_port": "9947",
+                "validator": true
+            },
+            {
+                "name": "bob",
+                "validator": true
+            },
+            {
+                "name": "charlie",
+                "validator": true
+            },
+            {
+                "name": "dave",
+                "validator": true
+            }
+        ]
+    },
+    "parachains": [
+        {
+            "id": 2000,
+            "chain_spec_path": "specs/single-container-template-container-2000.json",
+            "collators": [
+                {
+                    "name": "FullNode-2000",
+                    "validator": false,
+                    "command": "../target/release/container-chain-simple-node",
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"],
+                    "ws_port": 9949,
+                    "p2p_port": 33049,
+                    "prometheus_port": 33102
+                },
+                {
+                    "name": "Collator-01",
+                    "command": "../target/release/tanssi-node solo-chain",
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"],
+                    "prometheus_port": 33102
+                },
+                {
+                    "name": "Collator-02",
+                    "command": "../target/release/tanssi-node solo-chain",
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"],
+                    "prometheus_port": 33102
+                },
+                {
+                    "name": "Collator-03",
+                    "command": "../target/release/tanssi-node solo-chain",
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"],
+                    "prometheus_port": 33102
+                },
+                {
+                    "name": "Collator-04",
+                    "command": "../target/release/tanssi-node solo-chain",
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"],
+                    "prometheus_port": 33102
+                },
+                {
+                    "name": "Collator-05",
+                    "command": "../target/release/tanssi-node solo-chain",
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"],
+                    "prometheus_port": 33102
+                },
+                {
+                    "name": "Collator-06",
```

Comment:
You can remove some collators to make the test startup faster, I believe the default is 2 collators per container chain?

Reply:
Updated here https://github.com/moondance-labs/tanssi/pull/723/commits/d263ed4eba8e7bfe55da406d567248f45b882f46